p2p/simulations/adapters/types.go

```diff
@@ -265,3 +266,28 @@ func RegisterServices(services Services) {
 		os.Exit(0)
 	}
 }
+
+func (n *NodeConfig) initDefaultEnode() error {
+	return n.initEnode(net.IPv4(127, 0, 0, 1), 0, 0)
```

Comment:
For the simulation it doesn't matter. The transport layer is hacked anyway. And localhost:0 works with this hack.

Reply:
Is it fair to call it `Default` then? As I would expect something with default to work properly (not just in simulations).

Reply:
`Fallback`?

Reply:
I was not familiar enough with the code to come up with names before, so I spent some time on it. 

`Fallback` could work for me if in `NewNode()` functions this would not be our 1st pick.

What do you think about `initDummyEnode()`?

Reply:
I think `Default` covers it, because it indeed is the default configuration of a simnode. But I'm happy to call it just about anything :)